README.md

```diff
@@ -64,8 +64,10 @@ OpenLoco requires original files of Chris Sawyer's Locomotion to play. It can be
 ### Linux / macOS:
 - cmake
 - make or ninja
+
+32Bit versions of the following:
```

Comment:
`32-bit`. I'd prefer this to be an item with an indented list, like how the Visual Studio list is formatted.